src/app/blog/qad/page.mdx

```diff
@@ -0,0 +1,209 @@
+import { BlogPostLayout } from '@/components/BlogPostLayout'
+import {ThemeImage} from '@/components/ThemeImage'
+
+export const post = {
+  draft: false,
+  author: 'Floris Bruynooghe',
+  date: '2025-08-26',
+  title: 'Moving from STUN to QUIC Address Discovery',
+  description:
+    "Moving STUN into QUIC",
+}
+
+export const metadata = {
+    title: post.title,
+    description: post.description,
+    openGraph: {
+      title: post.title,
+      description: post.description,
+      images: [{
+        url: `/api/og?title=Blog&subtitle=${post.title}`,
+        width: 1200,
+        height: 630,
+        alt: post.title,
+        type: 'image/png',
+      }],
+      type: 'article'
+    }
+}
+
+export default (props) => <BlogPostLayout article={post} {...props} />
+
+# Holepunching
+
+As you probably know, iroh is in the business of holepunching and gives you a QUIC connection on top.
+The typical scenario is establishing a direct connection between two devices, like laptops or phones, both on different home networks.
+Home networks tend to have a [NAT] router in front of them,
+and, even when using IPv6, tend to block new incoming connections in the same fashion as a NAT router would.
+And to be fair, blocking random incoming connections to a home network is a sensible choice.
```

Comment:
```suggestion
As you probably know, `iroh` is in the business of holepunching.
The typical scenario is establishing a direct QUIC connection between two devices, like laptops or phones, both on different home networks.
Home networks tend to have a [NAT] router in front of them,
and tend to block new incoming connections even when using IPv6.
To be fair: blocking random incoming connections to a home network is a sensible choice.
```

"`iroh` is in the business of holepunching." Is just too good. I think it should be highlighted on it's own.

Reply:
I haven't done a full sweep of this blog yet, but when we refer to iroh in the blog (not at the start of the sentence), we've been using `iroh`. I don't catch them all, sometimes.